pod/perlsub.pod

```diff
@@ -445,8 +445,77 @@ If there are multiple optional positional parameters and not enough
 arguments are supplied to fill them all, they will be filled from left
 to right.
 
-After positional parameters, additional arguments may be captured in a
-slurpy parameter.  The simplest form of this is just an array variable:
+Since Perl 5.43.3, a signature may also declare named parameters.  Like
+positional parameters, these are lexical variables which will contain values
+passed by the caller when the subroutine is invoked.  Whereas positional
+parameters take their values from the corresponding position in the list
+passed by the caller, named parameters are matched by name-value pairs
+from the caller, where the name matches the variable name of the parameter
```

Comment:
I think , (the comma) is not needed here before "where".

Reply:
It probably reads better as a separate sentence even.